Sources/Sentry/include/HybridPublic/SentrySwizzle.h

```diff
@@ -83,6 +83,21 @@
         _SentrySWWrapArg(SentrySWArguments), _SentrySWWrapArg(SentrySWReplacement),                \
         SentrySwizzleMode, key)
 
+#if TEST || TESTCI
+/**
+ * Unswizzles the instance method of the class.
```

Comment:
`m`: Please also add a comment that we don't want to use this in production and only for tests explaining that this isn't safe to use in production.